app/services/decisions/case_decision_tree.rb

```diff
@@ -128,8 +128,6 @@ def blank_date_required?
     end
 
     def submission_root_step
```

Comment:
Likewise here, we can put `edit('/steps/submission/more_information')` under `when :upload_finished` directly in the decision tree